slack/ecs-deployment-failure/README.md

```diff
@@ -0,0 +1,31 @@
+# Terraform Module: Slack message for ECS Deployment Failure
+
+Post a message to Slack when the deployment is a failure. ECS Service need to enable the [deployment circuit breaker](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/deployment-circuit-breaker.html) in order to trigger the slack message.
+
+## Usage
+
+```terraform
+module "ecs_deployment_failure" {
+  source = "github.com/dbl-works/terraform//slack/ecs-deployment-failure"
+
+  project     = local.project
+  environment = local.environment
+
+  slack_webhook_url = "https://hooks.slack.com/services/xxxxxxxxx/xxxxxxxxxxx/yyyyyyyyyyyyyyyyyyyyyyyy"
+
+  # Optional
+  runtime = "ruby2.7" # Only can use ruby 3.2 after bumping the aws version to >5
+  timeout = 10 # timeout for lambda function
+  memory_size = 1024 # memory size for lambda function
```

Comment:
```suggestion
  memory_size = 128 # memory size for lambda function
```

⚠️ 1 GB of memory for a simple script is insane. People will copy this and turn off their brains. Let's default to `128` MiB which should be more than enough.